### ISSUE TITLE
Weird, inexplicable crashes when paired with a mutator called "Old School Zeds"

### ISSUE THREAD

**Issue by MortimerScharf** [poosh/KF-ScrnBalance#22]
Hello there!

So, the reason why I call these crashes "weird" and "inexplicable" is because they don't really make any sense. They happen at random, they can't be reliably recreated, and they seem to happen whenever they please. You could be playing five matches in a row and never experience it, or have it happen to you on wave 1 mid-game while killing a ZED (or even during tradertime!) 

Also, the mutator causes the server to crash during mapvote, either during the mapvoting itself or during map transitions. Thankfully, I have a script to bring the server back online, but still... it's very annoying.

I am running the latest version of ScrN. I am also running the latest version of this mutator (Old School Zeds), which can be found in the workshop. I already made a server from scratch to rule out any corrupted files, but the crashes are still happening. Here I am leaving several pastelinks with the logfiles. You can usually find the exact lines of the crash occuring at the end of each log. Worth noting is that I am running this server from my very own PC.

Pastebing logs:

https://pastebin.com/JPjqSVi9

https://pastebin.com/PZFnV6gn

https://pastebin.com/4QHBdf4Z

https://pastebin.com/rhQTF6gr

https://pastebin.com/RzBr3KXw

https://pastebin.com/ubVWT8C9

https://pastebin.com/7teutbGm

https://pastebin.com/ejs1qUBZ

https://pastebin.com/00YtnHaP

https://pastebin.com/bteuJvK1

https://pastebin.com/mVeGTCP1

https://pastebin.com/khxp9k7e

https://pastebin.com/yb5eu7Fb

https://pastebin.com/Jf6WSY64

https://pastebin.com/nxPicr79

https://pastebin.com/NaQ2KD7x

https://pastebin.com/MMNwYY4W

https://pastebin.com/3eHwXUy5

https://pastebin.com/rup6Xbp0

https://pastebin.com/M9413H4w

That's 20 logs, and I have many more but I can't paste them all due to Pastebin preventing me from posting more than 20 logs a day. Worth mentioning is that ScrN itself hasn't crashed a SINGLE time on me. Not even once (without the "Old School Zeds" mutator, that is). The moment I use the mutator... the problems start. I stopped using this mutator for a while because I got quite pissed at the crashes. I started a new server not too long ago, vanilla, without a single crash (so far), but the moment I try bringing in ScrN... the crashes started again. I am attaching the log from this "new" server with ScrN and OldSchoolZeds.

tl;dr: HELP!
[KFMOD-vie.011-11-2022_15.59.log](https://github.com/poosh/KF-ScrnBalance/files/9992963/KFMOD-vie.011-11-2022_15.59.log)

**Comment by poosh** [comment]
Does the server crash only when using ScrN + Old School Zeds? Have you tried running the stock ScrN Bundle without custom mutators?
Also, I'm unsure if `KFNoDramaMut` is compatible with ScrN Balance. Try disabling KFNoDramaMut and see if that helps.

**Comment by MortimerScharf** [comment]
Yeah, it ONLY crashes when I use ScrN + Old School Zeds. ScrN on its down crash at all, and OSZ doesn't crash either.

Alright, I will do all those tests as soon as I get home. I will remove NoDramaNut (which hasn't caused any crashed when using it with ScrN). Thanks, PooSH.

**Comment by MortimerScharf** [comment]
Alright, I left the server running (I am away for the time being) and I did exactly as you asked me: no NoDramaMut, no other mutators except for base Scrn Bundle... and it still crashed in the same way.

I don't really have access to my logs right now, but users have told me they experienced a crash on wave 6 while playing in the same way as all those other crashes have occurred.

**Comment by MortimerScharf** [comment]
Sorry for the THIRD POST in a row, but I just got home. Here's the .log file of when the server crashed while only running ScrN with the OldSchoolZeds mutator, no extra mutators or anything save for "Country Tags".
[KFMOD-sáb.012-11-2022_14.50.log](https://github.com/poosh/KF-ScrnBalance/files/10002823/KFMOD-sab.012-11-2022_14.50.log)

**Comment by poosh** [comment]
Did you try running the game without the OldSchoolZeds mutator?

**Comment by MortimerScharf** [comment]
Yeah, it never crashes. Just running ScrN + all those mods from before (and even the NoDramaMut) causes no crashes at all and I haven't experienced a single crash using JUST ScrN. OldSchoolZeds is causing the crashes.

**Comment by poosh** [comment]
Closed because the issue is not inside ScrN Balance
